WcaOnRails/config/application.rb

```diff
@@ -73,5 +73,8 @@ class Application < Rails::Application
     config.middleware.use Middlewares::WardenUserLogger, logger: ->(s) { Rails.logger.info(s) }
 
     config.autoload_paths << Rails.root.join('lib')
+
+    config.action_controller.asset_host = ENVied.ROOT_URL
```

Comment:
This is surprising to me. Is this needed to get the `self.avatar.url` method to return a url instead of a path? If so, that seems like a bug with the avatar code (returning a path when we're asking for a url).

I guess I broke this in https://github.com/thewca/worldcubeassociation.org/commit/cc770fe23c27acec18b749f12864d7e7dc731081?